web/src/settings_data.ts

```diff
@@ -351,3 +351,7 @@ export function get_request_data_for_stream_privacy(selected_val: string): {
         }
     }
 }
+
+export function guests_can_access_all_other_users(): boolean {
+    return user_groups.is_user_in_setting_group(realm.realm_can_access_all_users_group, 13);
```

Comment:
This is not correct. The correct way to test here is whether the realm_can_access_all_users_group is set to "Everyone" group or not.